src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -209,13 +204,26 @@ private Task WriteFormattedInt32Async(int value, CancellationToken cancellationT
                     request.Headers.TransferEncodingChunked = true;
                 }
 
+                if (request.Version.Major != 1)
+                {
+                    // TODO #23134: Support 2.0
+                    throw new PlatformNotSupportedException(SR.net_http_unsupported_version);
```

Comment:
I thought we didn't want to throw here.  David wrote "WinHttpHandler will use 1.1 for any unrecognized HttpRequestMessage.Version field.", and it looks like we similarly just let libcurl use its default in CurlHandler if the version isn't recognized: https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs#L522-L526

Reply:
I would prefer we don't throw.  We should match the behavior I described.  It's fine if you want to do some ETW logging here though.

Reply:
ok, I can remove it completely. I think it was there as a reminder but we can track it via issue. 
Falling back to 1.1 is probably good default.